runtime/fundamentals/ts_support.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "TypeScript support"
+oldUrl:
+  - /runtime/manual/advanced/typescript/
+  - /runtime/manual/typescript/
+  - /runtime/manual/typescript/overview/
+  - /runtime/manual/getting_started/typescript/
+  - /manual/advanced/typescript
+  - /runtime/manual/advanced/typescript/overview/
+  - /runtime/manual/advanced/typescript/types/
+---
+
+TypeScript is a first class language in Deno, just like JavaScript or
+WebAssembly, so you can run or import TypeScript without installing anything
+more than the Deno CLI.
+
+## The TS Compiler
+
+Deno has a built-in TypeScript compiler, which is used to compile TypeScript
+code to JavaScript. No extra installation or config is required making your
+development process simpler.
```

Comment:
I feel like we're repeating ourselves a bit: "...you can run (...) TypeScript..." "Deno has a built-in TypeScript compiler, which is used to compile TypeScript code to JavaScript".